Source/OxyPlot.SkiaSharp/SkiaRenderContext.cs

```diff
@@ -0,0 +1,762 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="SkiaRenderContext.cs" company="OxyPlot">
+//   Copyright (c) 2020 OxyPlot contributors
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace OxyPlot.SkiaSharp
+{
+    using global::SkiaSharp;
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    /// <summary>
+    /// Implements <see cref="IRenderContext" /> based on SkiaSharp.
+    /// </summary>
+    public class SkiaRenderContext : IRenderContext, IDisposable
+    {
+        private SKPaint paint = new SKPaint();
+        private SKPath path = new SKPath();
+        private float dpiScale = 1;
+
+        /// <inheritdoc/>
+        public bool RendersToScreen { get; set; } = true;
+
+        /// <summary>
+        /// Gets or sets the <see cref="SKCanvas"/> the <see cref="SkiaRenderContext"/> renders to. This must be set before any draw calls.
+        /// </summary>
+        public SKCanvas SkCanvas { get; set; }
```

Comment:
Could this be set in a constructor to improve discoverability and minimise the opportunity to forget setting it? (Other renders accept a `null` canvas for the purposes of text measuring, which I'd want to be explicitly acknowledged as in `Oxyplot.Wpf`.

Reply:
Setting this via the constructor would be the obvious choice when we are only considering exporters.
With potential `PlotView` implementations however it's a slightly different story: we don't have the `SKCanvas` at the time of creating the `SkiaRenderContext`, rather we get it passed from the `OnRender` (or similar) event - potentially a different `SKCanvas` object every time. So in any case we need the setter.
We could initialize this from the constructor, but allow `null` to be passed.

Reply:
> We could initialize this from the constructor, but allow null to be passed.

That's what `CanvasRenderContext` and `GraphicsRenderContext` do, but if you reckon the setter is necessary than I'd be happy leaving it as is.